packages/express-api-reference/README.md

```diff
@@ -15,35 +15,18 @@ npm install @scalar/express-api-reference
 
 ## Usage
 
-[Set up Express](https://expressjs.com/en/starter/hello-world.html) and pass an OpenAPI/Swagger spec to the `apiReference` middleware:
+[Set up Express](https://expressjs.com/en/starter/hello-world.html) and pass an URL to an OpenAPI/Swagger document to the `apiReference` middleware:
```

Comment:
I think this should be `a`

```suggestion
[Set up Express](https://expressjs.com/en/starter/hello-world.html) and pass a URL to an OpenAPI/Swagger document to the `apiReference` middleware:
```

Reply:
Alright I looked it up and I'm not crazy, since URL starts with a consonant sound (**Y**ou-Are-Ell) it should be "a URL" not "an URL".

This article has a good section on acronyms https://www.grammar.com/a-vs-an-when-to-use

Reply:
I think this still needs to be an `a`